build/commands/lib/gitPatcher.test.js

```diff
@@ -26,6 +26,11 @@ function getPatch (gitRepoPath, modifiedFilePath) {
 }
 
 describe('Apply Patches', function () {
+  // Some CI nodes have trouble with these tests, occassionally taking multiple
+  // seconds for one of the simple commands. We increase the timeout to account
+  // for that.
+  jest.setTimeout(20000)
```

Comment:
I'd rather increase this globally to be 30 seconds similar to browser/unit tests. Don't think we want to nitpick similar tests in future.

Reply:
updated